CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: Using the `current_timestamp` in generated columns doesn't result in
```

Comment:
isn't the bug, that if you use a current_timestamp in generated colum, the insert will fail at all?